.travis.yml

```diff
@@ -23,7 +23,7 @@ script:
   - if [ $ARMADILLO == "latest" ]; then
       curl https://ftp.fau.de/macports/distfiles/armadillo/`curl https://ftp.fau.de/macports/distfiles/armadillo/ -- | grep '.tar.xz' | sed 's/^.*<a href="\(armadillo-[0-9]*.[0-9]*.[0-9]*.tar.xz\)".*$/\1/' | tail -1` | tar xvJ && cd armadillo*;
     else
-      curl https://ftp.fau.de/macports/distfiles/armadillo/armadillo-8.400.0.tar.xz | tar -xvJ && cd armadillo*;
+      curl https://ftp.fau.de/macports/distfiles/armadillo/armadillo-9.800.1.tar.xz | tar -xvJ && cd armadillo*;
```

Comment:
Suggest to use 9.800.6 instead, directly from Sourceforge:
https://sourceforge.net/projects/arma/files/armadillo-9.800.6.tar.xz
the .6 patch release is the last release in the 9.800.x series.

I'm not going to delete the above file (until at least when ensmallen updates its minimum armadillo version), so it's safe to use.  The previous situation which resulted in old armadillo releases being removed no longer applies.